src/main/java/duke/parser/Parser.java

```diff
@@ -24,11 +24,15 @@
     private static final Pattern GENERAL_COMMAND_FORMAT =
             Pattern.compile("(?<commandType>\\S+)(?<arguments>.*)");
 
+    private static final String LIST_COMMAND = "list";
+    private static final String EXIT_COMMAND = "bye";
+    private static final String EXPORT_LOCKER_COMMAND = "export";
+    private static final String EXPORT_SELECT_COMMAND = "exports";
```

Comment:
if i am not wrong i removed all this in my latest PR. The one about code cleaning

Reply:
removed and cleaned